src/lib.rs

```diff
@@ -1520,6 +1532,19 @@ impl<'a> RenderPass<'a> {
     ///
     /// The active index buffer can be set with [`RenderPass::set_index_buffer`], while the active
     /// vertex buffers can be set with [`RenderPass::set_vertex_buffer`].
+    /// 
+    /// The structure expected in `indirect_buffer` is the following:
+    /// 
+    /// ```rust
+    /// #[repr(C)]
+    /// struct DrawIndexedIndirect {
+    ///     vertex_count: u32,
+    ///     instance_count: u32,
+    ///     first_index: u32,
+    ///     base_vertex: u32,
```

Comment:
This one needs to be `i32`. The OpenGL documentation you linked appears to be wrong.
See https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VkDrawIndexedIndirectCommand.html for example.

Reply:
I'm not even surprised the opengl docs are wrong.